### PR TITLE
Let's make this more fun!

### PR DESCRIPTION
**Pull request by cmertbjp**

### CODE REVIEW

**Comment by lidalaine** [comment]
何！！

File: index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
 function say_hi() {
     var fname = document.getElementById('first_name').value;
  
-    var html = 'Привет, ' + fname + '!' ;
+    var html = 'Привет, ' + fname + '! お元気ですか？' ;
```
Comment (lidalaine) [inline index.html:17]: Change it

**Review by lidalaine — COMMENTED** [review]
Onegai

File: index.html (hunk 1 above)
Comment (cmertbjp) [reply index.html:17]: change what to what?

Comment (lidalaine) [reply index.html:17]: Bring it back!

Comment (cmertbjp) [reply index.html:17]: it's already back, it's still on my private branch

**Review by lidalaine — APPROVED** [review]

**Pull request merged**